tests/client.rkt

```diff
@@ -1,14 +1,16 @@
-#lang racket/base
+#lang racket
```

Comment:
`racket/base` is very desirable for libraries. Since this is a test, it isn't a big deal, but FYI

Reply:
ok